website/docs/components/icon/partials/guidelines/overview.md

```diff
@@ -0,0 +1,6 @@
+## What is an icon
+
+An icon is used to provide visual flourish to text and, in some rare contexts, meaning on its own.
```

Comment:
@alex-ju @didoo Maybe this is a better way to phrase this?
@hashicorp/hds-design some input on this would be helpful. 

```suggestion
An icon is used to provide additional meaning to text and, in some rare contexts, meaning on its own.
```

Reply:
+1, icons aren't often used purely for visual flourish.

Reply:
+1 on the flourish language, another note: we explicitly say that icons shouldn't be used on their own/in isolation, but here we're reference the icon having it's own meaning. Is that something that should be excluded or will consumers be confused by this?